.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
its nice to add prettier, but thats unrelated to `protect against undefined`

Please remove these changes and make a separate PR